test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -599,7 +599,7 @@ def test_dtensor_dont_recompile_on_same_placement_devicemesh(self):
 
         @torch.compile(backend=cnt)
         def fn(x):
-            dt = DTensor.from_local(x, mesh, [placement], run_check=False)
+            dt = DTensor.from_local(x, mesh, [placement], run_check=False)  # noqa: F841
```

Comment:
cc @jansel given that this is not returned is it going to be DCEd and this test is a bit moot? What is the right fix here?